app/views/directives/create-secret.html

```diff
@@ -47,268 +47,296 @@
           Unique name of the new secret.
         </div>
     </div>
-    <div class="form-group">
-      <label for="authentificationType">Authentication Type</label>
-      <ui-select required ng-model="newSecret.authType" search-enabled="false">
-        <ui-select-match>{{$select.selected.label}}</ui-select-match>
-        <ui-select-choices repeat="type.id as type in secretAuthTypeMap[newSecret.type].authTypes">
-          {{type.label}}
-        </ui-select-choices>
-      </ui-select>
-    </div>
 
-    <div ng-if="newSecret.authType === 'kubernetes.io/basic-auth'">
+    <div ng-if="newSecret.type == 'webhook'">
       <div class="form-group">
-        <label for="username">Username</label>
-        <div>
+        <label for="webhookSecretKey" class="required">Webhook Secret Key</label>
+        <div class="input-group">
           <input class="form-control"
-            id="username"
-            name="username"
-            ng-model="newSecret.data.username"
+            id="webhookSecretKey"
```

Comment:
nit: we usually kebab case ids (webhook-secret-key)

Reply:
Need to check all id/for attributes as most of them are camel and not kebab.